content/projects/gsoc/2023/project-ideas/displaying_plugin_health_scores.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: gsocprojectidea
+title: "Displaying of Plugin Health Scores"
+goal: "Display a possibly filtered view of Plugin Health Scores on plugins.jenkins.io and the details for each individual plugin."
+category: Tools
+year: 2023
+status: discussion
+sig: platform
+skills:
+- Java
+- YAML
+- Command line tools
+- Package management tool theory
+mentors:
+- "jleon"
```

Comment:
Hi @jmMeessen @jleon33 @kwhetstone @stopalopa @timja
Is `jleon` the correct?
My suggestion is to change it to `jleon33`
Additionally:
https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc/edit
also lists @kwhetstone @stopalopa @timja as potential mentors
Hence my suggestion is to amend it as:
```suggestion
jleon33
kwhetstone
stopalopa
timja
```

Reply:
Hello @lloydchang 
Thank you for looking closely at this PR.

Please note that the mentor name set in the adoc template references the jenkins.io author list (and not the GitHub handle as @jleon33 ). `jleon` points to https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/authors/jleon.adoc and is thus correct.

Additionally, the document you are referencing is a placeholder draft description in the commented out section. It points to an outdated and different project idea. 
Unless changes that I am not aware off, neither @kwhetstone , @stopalopa , nor @timja volunteered for GSoC2023 mentorship. 

This means that you update proposal is not acceptable. But thanks to have looked into this.